README.md

```diff
@@ -255,11 +255,44 @@ By default, all files in the S3 directory are uploaded. When the `dry_run` input
 is set to anything other than `false`, no files are uploaded, but instead the
 filename along with the resulting location in the bucket is printed.
 
+## Create Release Branch
```

Comment:
Wasn't this already added by #59?

Reply:
Drat, forgot to create a new branch, reverting.

Reply:
Okay, pulling from main resolved it.